mdpow/tests/test_analysis_alchemlyb.py

```diff
@@ -62,10 +64,15 @@ def fep_benzene_directory(tmpdir_factory):
 class TestAnalyze(object):
     def get_Gsolv(self, pth):
         gsolv = pth.join("FEP", "water", "Gsolv.fep")
-        G = pickle.load(gsolv.open())
-        # patch paths
+        if sys.version_info.major == 3:
+            with open(gsolv, 'rb') as f:
+                G = pickle.load(f, encoding='latin1')
+                # patch paths
+        elif sys.version_info.major == 2:
+            G = pickle.load(gsolv.open())
```

Comment:
see above comment

Reply:
Added comment needed to read pickle files from python 2

Reply:
Can you please add the comment right here near the code?

Reply:
Added a comment